### PR TITLE
Fix logger and return C example

### PR DESCRIPTION
**Pull request by c-solo**
- Adds possibility to skip examples.
- Returns C example
- Uncomment Rust-regex and go-simple examples
- Merge fix for late init for logger

### CODE REVIEW

File: examples/c-simple/src/lib.c [hunk 1]
```diff
@@ -0,0 +1,3 @@
+int addOne(int x) {
+  return x + 1;
+}
```
Comment (mikevoronov) [inline examples/c-simple/src/lib.c:3]: eof

File: examples/c-simple/src/main/java/asmble/examples/csimple/Main.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+import asmble.generated.CSimple;
+
+class Main {
+    public static void main(String[] args) {
+        // Doesn't need memory or method table
+        CSimple simple = new CSimple(0, new MethodHandle[0]);
+        System.out.println("25 + 1 = " + simple.addOne(25));
+    }
+}
```
Comment (mikevoronov) [inline examples/c-simple/src/main/java/asmble/examples/csimple/Main.java:13]: ditto

File: examples/c-simple/src/main/java/asmble/examples/csimple/Main.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package asmble.examples.csimple;
+
+import java.lang.invoke.MethodHandle;
+
+import asmble.generated.CSimple;
+
+class Main {
+    public static void main(String[] args) {
+        // Doesn't need memory or method table
```
Comment (mikevoronov) [inline examples/c-simple/src/main/java/asmble/examples/csimple/Main.java:9]: It is better to add It

**Review by mikevoronov — APPROVED** [review]

File: examples/c-simple/src/lib.c (hunk 1 above)
Comment (c-solo) [reply examples/c-simple/src/lib.c:3]: Will add in next PR

**Pull request merged**